cdist/conf/type/__rm/man.rst

```diff
@@ -0,0 +1,52 @@
+cdist-type__rm(7)
+=================
+
+NAME
+----
+cdist-type__rm - Remove files and directories which match the pattern.
+
+
+DESCRIPTION
+-----------
+This type will DELETE your files and directories which match the pattern.
+
+Patterns are passed to ``find``'s ``-path`` - see find(1) for more details.
+
+
+REQUIRED PARAMETERS
+-------------------
+remove
+   Remove pattern.
```

Comment:
Can you better explain what "Remove pattern" is? Even rename the parameter?

Reply:
rename to what? at first i used `--pattern`, but it didn't feel right :disappointed:

Reply:
Is this parameter's value path pattern or file (inside path) pattern?
Depending on that it could be named as `remove-file-pattern`/`remove-path-pattern`.
I am not sure, but `remove` is confusing.

Reply:
then `--pattern`? should be clear enough and nicely short.

Reply:
Fine with me, better than remove.